json/players/osla.json

```diff
@@ -7,7 +7,5 @@
     		"BULLETS_PLAYER" : 0.5,
 		"BULLETS_ENEMY" : 0.0,
 		"LOOT" : 0.5
-	},
```

Comment:
While you're at it, it would be good to remove the ColtExpress specific points here - the only line needed is the ScoreHeuristic